roles/ups/templates/prometheus_rule.yml.j2

```
@@ -0,0 +1,144 @@
+# Monitor Service (Metrics)
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    monitoring-key: middleware
+    prometheus: application-monitoring
+    role: alert-rules
+  name: unifiedpush
+spec:
+  selector:
+    matchLabels:
+      app: unifiedpush
+  groups:
+    - name: general.rules
+      rules:
+      - alert: UnifiedpushDown
+        expr: absent(kube_pod_container_status_running{namespace="{{ ups_namespace }}",container="ups"}==1)
+        for: 5m
+        labels:
+          severity: critical
+        annotations:
+          description: "The UnifiedPush Pod Server has been down for more than 5 minutes. "
+          summary: "The UnifiedPush Pod Server is down. For more information see on the UnifiedPush at https://github.com/aerogear/aerogear-unifiedpush-server"
+          sop_url: "https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-push.adoc" #todo
+      - alert: UnifiedPushConsoleDown
+        expr: absent(kube_endpoint_address_available{endpoint="example-unifiedpushserver-unifiedpush"} == 1)
+        for: 5m
+        labels:
+          severity: critical
+        annotations:
+          description: "The UnifiedPush admin console has been down for more than 5 minutes. "
+          summary: "The UnifiedPush admin console endpoint has been unavailable for more that 5 minutes. For more information see on the UnifiedPush  at https://github.com/aerogear/UnifiedPush "
+          sop_url: "https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-push.adoc" #todo
+      - alert: UnifiedPushDatabaseDown
+        expr: absent(kube_pod_container_status_running{namespace="{{ ups_namespace }}",container="postgresql"}==1)
+        for: 5m
+        labels: 
+          severity: critical
+        annotations:
+          description: "The UnifiedPush Database pod has been down for more than 5 minutes"
+          summary: "The UnifiedPush Database is down. For more information see on the UnifiedPush at https://github.com/aerogear/aerogear-unifiedpush-serve"
+          sop_url: "https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-push.adoc" #todo
+      - alert: UnifiedPushPodCPUHigh # todo check if it will work after expse the metrcis
+        expr: "(rate(process_cpu_seconds_total{job='example-unifiedpushserver-unifiedpush'}[1m])) > (((kube_pod_container_resource_limits_cpu_cores{namespace='{{ ups_namespace }}',container='application'})/100)*90)"
```

```suggestion
        expr: "(rate(process_cpu_seconds_total{job='example-unifiedpushserver-unifiedpush'}[1m])) > (((kube_pod_container_resource_limits_cpu_cores{namespace='{{ ups_namespace }}',container='ups'})/100)*90)"
```

I presume that's the expected container at least...there isn't one called 'application' as far as I know. Also, it looks like this might be expecting cpu resource limits to be set on the container (I might be wrong about that), which we don't appear to do: https://github.com/aerogear/unifiedpush-operator/blob/master/pkg/controller/unifiedpushserver/unifiedpushserver.go#L176..L256

---

@grdryn yeah, it is a copy/paste of the existing [alert file](https://github.com/aerogear/unifiedpush-operator/blob/master/deploy/monitor/push_prometheus_rule.yaml). Maybe @camilamacedo86 can answer the question?